assets/css/src/components-form-editor/_form-placement.scss

```diff
@@ -61,5 +61,6 @@
 }
 
 .form-editor-placement-selection {
+  font-size: 14px;
```

Comment:
We have a set of variables for font sizes. Take a look at https://github.com/mailpoet/mailpoet/blob/master/assets/css/src/settings/_typography.scss
Could you please use one of them?

Reply:
Of course